### ISSUE TITLE
Inserting blank lines before an image, after inserting the image, removes image when sent.

### ISSUE THREAD

**Issue by ckrzen** [tutao/tutanota#1515]
Inserting blank lines before an image, after inserting the image, removes image when sent.

I noticed that when I type a few words into the main body of an email and then go to the toolbar button: "Insert Image" that the image is in-line and as an attachment in the header area. If I then move the cursor to end-of-line in the words at beginning of main body text and insert LF or CR newlines, that the image remains visible in-line, but is removed from the attachment header and is not sent with the email.

Images should stay attached when adding newlines before insert point.

BEFORE LF/CR NEWLINES ADDED(screenshot):

![Screenshot from 2019-09-26 15-43-07](https://user-images.githubusercontent.com/6942277/65723761-b04a5380-e074-11e9-9374-db867b0253dd.png)

AFTER LF/CR NEWLINES ADDED(screenshot):

![Screenshot from 2019-09-26 15-46-53](https://user-images.githubusercontent.com/6942277/65724041-336ba980-e075-11e9-81ad-caf9325246c4.png)

Fedora Linux Workstation 30
Firefox Quantum 69.0.1
GNOME 3.32

**Comment by charlag** [comment]
I checked it and it seems like Squire removes and inserts elements again (for some reason). We could improve MutationObserver here https://github.com/tutao/tutanota/blob/ada0750ff4bd9de41dc4558df0afbeedd0dcbce3/src/mail/MailEditor.js#L591-L605
to check if the image is inserted again. I tried it out and it was not that simple because it is inserted deeper than the top level of `addedNodes`. Searching through all the changed nodes in this case doesn't seem right, maybe there's some better way or maybe Squire could stop doing that.
